kotlinx-coroutines-debug/src/CoroutinesBlockHoundIntegration.kt

```diff
@@ -77,9 +77,6 @@ public class CoroutinesBlockHoundIntegration : BlockHoundIntegration {
         for (method in listOf("clear", "peek", "removeFirstOrNull", "addLast")) {
             allowBlockingCallsInside("kotlinx.coroutines.internal.ThreadSafeHeap", method)
         }
-        // [addLastIf] is only used in [EventLoop.common]. Users of [removeFirstIf]:
-        allowBlockingCallsInside("kotlinx.coroutines.test.TestCoroutineDispatcher", "doActionsUntil")
-        allowBlockingCallsInside("kotlinx.coroutines.test.TestCoroutineContext", "triggerActions")
```

Comment:
Not directly related, but looking at this, I don't know how I feel about it being forbidden to do the test module's `runCurrent` and `advance*` from `Dispatchers.Default` when BlockHound is used, even if the operations themselves are completely fine. I don't think this is a realistic concern, but it still feels a bit semantically unclean.